build/templates/grpc_session_options.py.mako

```diff
@@ -41,24 +45,34 @@ class SessionInitializationBehavior(IntEnum):
 class GrpcSessionOptions(object):
     '''Collection of options that specifies session behaviors related to gRPC.'''
 
-    def __init__(self, grpc_channel, session_name, initialization_behavior=SessionInitializationBehavior.AUTO):
+    def __init__(
+        self,
+        grpc_channel,
+        session_name,
+        *,
+        api_key=MEASUREMENT_LINK_23Q1_NIMI_PYTHON_API_KEY,
```

Comment:
[10:42 AM] John Bongaarts
It's `MEASUREMENTLINK_...`
Single word.

Reply:
Done